www/door/index.php

```diff
@@ -1,49 +1,73 @@
 <?php
+
 require_once dirname(dirname(__DIR__)) . implode(DIRECTORY_SEPARATOR, ['', 'inc', 'include.php']);
-$doors = new \pvv\side\Doors($pdo);
 
-$out = null;
 header('Content-Type: application/json');
-if ($_SERVER['REQUEST_METHOD'] === 'GET') {
-    if (isset($_GET["name"])) {
-        $out = $doors->getByName($_GET["name"]);
-        if (!$out) {
-            echo '{"error": true, "reason": "not found"}';
-            http_response_code(404);
-            exit();
+$door = new \pvv\side\Door($pdo);
+
+if($_SERVER['REQUEST_METHOD'] === 'POST') {
+    if (isset($_SERVER["HTTP_AUTHORIZATION"])) {
+        list($type, $data) = explode(" ", $_SERVER["HTTP_AUTHORIZATION"], 2);
+        if (strcasecmp($type, "Bearer") == 0) {
+            if (hash_equals($data, $doorSensorSecret)) {
```

Comment:
Where is `doorSensorSecret` set?

Reply:
In sql_config.php, appended a dummy in sql_config_example.php